assisted-inject-processor-dagger2/src/main/java/com/squareup/inject/assisted/dagger2/processor/internal/hilt.kt

```diff
@@ -0,0 +1,19 @@
+package com.squareup.inject.assisted.dagger2.processor.internal
+
+import com.squareup.inject.assisted.processor.internal.getAnnotation
+import com.squareup.javapoet.AnnotationSpec
+import com.squareup.javapoet.ClassName
+import javax.lang.model.element.TypeElement
+import javax.lang.model.util.Elements
+
+internal fun generateHiltAnnotation(moduleType: TypeElement, elements: Elements): AnnotationSpec? =
+  // If the @AssistedModule class also has an @InstallIn we just mirror it
```

Comment:
https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html#:~:text=A%20small%20one%20that%20is,end%20them%20with%20a%20period.

Same for the others.

Reply:
Fixed